docs/developer-guide/nodejs-api.md

```diff
@@ -852,6 +853,40 @@ Example of customizing `RuleTester`:
 ```js
 "use strict";
 
+const RuleTester = require("eslint").RuleTester,
+    test = require("my-test-runner"),
+    myRule = require("../../../lib/rules/my-rule");
+
+let title;
+
+class MyRuleTester extends RuleTester {
```

Comment:
the reason for this, is in https://github.com/jfmengels/eslint-ava-rule-tester/blob/master/index.js if someone calls:
```
const t = avaRuleTester(test, ...);
const t2 = avaRuleTester(test2, ...);
t.run(...)  // runs with test2 but should be test1
t2.run(...) // runs with test2
```
then unexpected tests get run